### PR TITLE
Feature/menu

### PR DESCRIPTION
**Pull request by kevgathuku**
@andela-gukpere Please review

### CODE REVIEW

File: app/styles/style.less [hunk 1]
```diff
@@ -253,14 +272,14 @@ img {
...
 .select {
-    display: inline-block;
-    width: 50%;
+  display: inline-block;
+  width: 50%;
 }
 
 .success-toast {
-    background-color: #00cc33;
+  background-color: #00cc33;
```
Comment (andela-gukpere) [inline app/styles/style.less]: **Code Quality - Nitpick**

You should use the shortened form of this color.
Colors can be shortened when they a grouped and double.
i.e `#00cc33` is `#0c3` and `#444444` is `#444`

File: app/index.jade [hunk 2]
```diff
@@ -6,3 +6,10 @@ html
     #ui-view
     script(src="./js/app.js")
     include shared/footer
+    script(type='text/javascript').
+      $(".button-collapse").sideNav(
```
Comment (andela-jkithome) [inline app/index.jade]: Remove jQuery from this file.

File: app/scripts/components/Login/AuthBtn.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+ var React = require('react');
+
+  module.exports = React.createClass({
+    render: function() {
+      return (
+        <div>
+          <p>or</p>
+          <div className="row">
+            <a href="/auth/github" className="waves-effect waves-light btn">
```
Comment (andela-jkithome) [inline app/scripts/components/Login/AuthBtn.jsx]: Add space between icon and button name.

Comment (andela-gukpere) [inline app/scripts/components/Login/AuthBtn.jsx]: **UI/UX - Nitpick**

There should be a single space (or margin) separating the icon from the text, for a neat look.

**Pull request merged**